x/posts/internal/keeper/keeper_test.go

```diff
@@ -47,6 +47,50 @@ func TestKeeper_GetLastPostId(t *testing.T) {
 	}
 }
 
+func TestKeeper_GetPostHashtags(t *testing.T) {
```

Comment:
Can we add a case to test identical hashtags used multiple times inside the same message to make sure that their values are extracted correctly? 

- Message: `#test post with multiple #test tags inside` 
- Expected result: [`test`]